.github/workflows/rust.yml

```diff
@@ -72,6 +72,11 @@ jobs:
         cd examples
         CARGO_TARGET_WASM32_UNKNOWN_UNKNOWN_RUNNER=../target/debug/test-runner cargo test --target wasm32-unknown-unknown
         cargo test --target x86_64-unknown-linux-gnu
+    - name: Check explorer schema
+      uses: actions/setup-node@v3
+      run: |
+        npm install get-graphql-schema
+        cargo test test_end_to_end_check_schema
```

Comment:
I think that gets executed with the other end-to-end tests anyway, so now it would run twice?
If we want to run it in a separate workflow, we should probably mark it `#[ignore]` and pass `--ignored` here. But maybe it's fine if it just runs together with the other tests?

Reply:
even with the `--exclude` argument?

Reply:
Ah, sorry, totally missed that!